src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -404,36 +386,30 @@ static LoaderResult _parseShapeProperty(tvgBlock block, Shape *shape)
             }
             break;
         }
-        default:
-        {
-            return LoaderResult::InvalidType;
-        }
+        default: return LoaderResult::InvalidType;
     }
-
     return LoaderResult::Success;
 }
 
-// Picture
+
 static LoaderResult _parsePicture(tvgBlock block, Picture *picture)
 {
-    switch (block.type)
-    {
-        case TVG_RAW_IMAGE_BEGIN_INDICATOR:
-        {
+    switch (block.type) {
+        case TVG_RAW_IMAGE_BEGIN_INDICATOR: {
             if (block.length < 2*sizeof(uint32_t)) return LoaderResult::SizeCorruption;
 
-            const char* pointer = block.data;
+            const char* ptr = block.data;
             uint32_t w, h;
```

Comment:
these are floats in PictureImpl, since the loader expects uints, I need to cast these floats in the saver.

Reply:
w and h are used for pixel data size, it must not be float. I dont get why w and h are float, not uint32_t, in `tvgLoader.h`